exporter/opentelemetry-exporter-jaeger/tests/test_jaeger_exporter_protobuf.py

```diff
@@ -71,11 +71,8 @@ def test_constructor_by_environment_variables(self):
 
         env_patch.start()
 
-        exporter = JaegerSpanExporter(
-            service_name=service, transport_format="protobuf"
-        )
+        exporter = JaegerSpanExporter(transport_format="protobuf")
 
-        self.assertEqual(exporter.service_name, service)
```

Comment:
Instead of removing this assert I would suggest patch the os.environ `OTEL_RESOURCER_ATTRIBUTES` and use that `service.name` here.

Reply:
patched the OTEL_RESOURCER_ATTRIBUTES